lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -92,6 +93,8 @@ UnsignedInteger LHSResult::getNumberOfRestarts() const
 /** Attributes for getting elements of result */
 Sample LHSResult::getOptimalDesign() const
 {
+  if (!collDesigns_.getSize())
+    throw InvalidArgumentException(HERE) << "No design stored!";
```

Comment:
Strange to invoke invalid arguments for a method with no argument! Probably InternalException is more relevant here